Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:8.12.0-alpine as development
+
+COPY ./.eslintrc.js /api/.eslintrc.js
+COPY ./package.json /api/package.json
+COPY ./server.js /api/server.js
```

Comment:
@uselessdev por que o eslintrc.js e server.js são copiados mas não são usados no Dockerfile?

Reply:
Não tenho certeza :thinking: técnicamente então eu não preciso copiar nenhum arquivo no Dockerfile se ele não for usado?

Reply:
se não for usado no Dockerfile não

Reply:
No caso do `package.json` faz sentido deixar ele no Dockerfile?